src/exchange/billing/account-renew/billing-account-renew.html

```diff
@@ -0,0 +1,136 @@
+<div class="px-5 pt-5" data-ui-view>
+    <oui-back-button data-on-click="$ctrl.goBack()">
+        <span data-translate="exchange_update_billing_action_title"></span>
+    </oui-back-button>
+
+    <oui-stepper
+        data-on-finish="submit()">
+        <oui-step-form
+            data-header="{{::'exchange_update_billing_button_title' | translate}}"
+            data-description="{{::'exchange_update_billing_periode_intro' | translate}}"
+            data-loading="!$ctrl.exchange"
+            data-on-cancel="$ctrl.goBack()"
+            data-valid="hasChanged()"
```

Comment:
```suggestion
            data-valid="$ctrl.hasChanged()"
```
?

Reply:
Same as `submit()`, it's exposed on the scope. But I could refactor it.